src/services/XmlSignerService.php

```diff
@@ -25,23 +27,23 @@ public static function signXml($xml, $certPath, $certPassword = '')
 
         // --- 2. Use XMLSecurityDSig and XMLSecurityKey from the xmlseclibs library ---
         // xmlseclibs: https://github.com/robrichards/xmlseclibs
-        if (!class_exists('XMLSecurityDSig')) {
+        if (!class_exists(XMLSecurityDSig::class)) {
```

Comment:
This will fail if class is not available. Please change it to the fully qualified class name as string.